net/apfree-wifidog/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2018 Dengfeng Liu
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apfree-wifidog
+PKG_VERSION:=3.8.1588
+PKG_RELEASE=1
+
+
+PKG_LICENSE:=GPL-3.0
+PKG_MAINTAINER:=Dengfeng Liu<liudf0716@gmail.com>
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/liudf0716/apfree_wifidog.git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

```

Comment:
Unnecessary when using PKG_SOURCE_PROTO. Actually the default is to use a .xz archive which is a lot smaller.

Reply:
@neheb ok, thanks for your advise.